app/frontend/src/features/map/MapPage.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import PageTitle from "../../components/PageTitle";
+import Map from "../../components/Map";
+import { userRoute } from "../../AppRoutes";
+import { LngLat, Map as MapboxMap } from "mapbox-gl";
+import { useHistory, useLocation } from "react-router-dom";
+
+import { addClusteredUsersToMap } from "./ClusteredUsers";
+
+export default function MapPage() {
+  const history = useHistory();
+
+  const location = useLocation();
+
+  const userClicked = (ev: any) => {
+    const username = ev.features[0].properties.username
```

Comment:
Is this documented in their docs or just a lot of trial and error to find this property/object exists on the event?

Reply:
The `features` (stuff on map) and `properties` (of features) stuff is GeoJSON terminology, and `username` is a property that every point gets from our backend in the returned GeoJSON.